02-microarray/pathway_analysis_microarray_06_ssgsea.Rmd

```diff
@@ -321,24 +321,24 @@ comp_coag_df <- metadata_df %>%
 
```

Comment:
Looks like `library(ggplot2)` needs to be added at the top of this notebook now as well.

Reply:
These notebooks haven't been updated at all, so I'll leave them as is and when we address #173 we can address that too.